worldedit-forge/src/main/java/com/sk89q/worldedit/forge/ForgeWorld.java

```diff
@@ -238,7 +238,7 @@ public boolean regenerate(Region region, EditSession editSession) {
                 try {
                     u = ChunkProviderServer.class.getDeclaredField("field_73248_b"); // chunksToUnload

```

Comment:
Possibly, however this works and is inline with the original code, i remember having issues with reflectionhelper screwing in the past.